src/dataDisplay/Number/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+const DEFAULT_SHOW_DECIMALS = true;
+const DEFAULT_SHOW_THOUSAND_SEPARATORS = false;
+
+type FormatNumberOptions = {
+  roundingType?: 'up' | 'down';
+  showDecimals?: boolean;
+  decimalPlaces?: number;
+  showThousandSeparators?: boolean;
+  currency?: string;
+  as?: 'p' | 'span';
+};
+
+export const formatNumber = (
+  value: string,
+  options: FormatNumberOptions
```

Comment:
`options` parameter is required, but all of the properties inside the object are optional. Should it be optional too?

Reply:
What do you think when I create a utility type that makes at least one required?

Reply:
Sorry, I'm not getting it. Why is one of them required? As I can see, there are default values for each of them except `currency`

Reply:
You would only use the component if you wanted to format a number, hence using at least one of the props. Otherwise you wouldn't use it at all. What are your thoughts?

Reply:
I'd set up the defaults parameters to be the same as we use in the Safe app:
https://github.com/gnosis/safe/blob/master/archive/specs/common/format_amounts.rst
https://github.com/gnosis/safe-react/blob/8b35069e15e970dd8e27c9d44f527e3cd001c4a1/src/logic/tokens/utils/formatAmount.ts